metrics/Score Metric.ipynb

```diff
@@ -0,0 +1,555 @@
+       " 'Ed-Fi-Alliance-OSS/.github',\n",
+       " 'Ed-Fi-Alliance-OSS/slsa-github-generator',\n",
+       " 'Ed-Fi-Alliance-OSS/Ed-Fi-DataImport',\n",
+       " 'Ed-Fi-Alliance-OSS/AdminAPI-1.x',\n",
+       " 'Ed-Fi-Alliance-OSS/Ed-Fi-Compatibility-Libraries',\n",
+       " 'Ed-Fi-Alliance-OSS/vscode-metaed-ide',\n",
+       " 'Ed-Fi-Alliance-OSS/antlr4',\n",
+       " 'Ed-Fi-Alliance-OSS/AdminAPI-2.x',\n",
+       " 'Ed-Fi-Alliance-OSS/Ed-Fi-API-Standards',\n",
+       " 'Ed-Fi-Alliance-OSS/api.ed-fi.org',\n",
+       " 'Ed-Fi-Exchange-OSS/Ed-Fi-TPDM-Community-Model',\n",
+       " 'Ed-Fi-Exchange-OSS/Ed-Fi-X-StudentEngagement',\n",
+       " 'Ed-Fi-Exchange-OSS/EWS-Starter-Kits',\n",
+       " 'Ed-Fi-Exchange-OSS/Ed-Fi-X-ODS-Docker',\n",
+       " 'Ed-Fi-Exchange-OSS/Suite-3-Performance-Testing',\n",
+       " 'Ed-Fi-Exchange-OSS/Ed-Fi-X-OtisEd-iMart',\n",
+       " 'Ed-Fi-Exchange-OSS/TPDM-Data-Ingestion-POC',\n",
+       " 'Ed-Fi-Exchange-OSS/Dashboard-Replacement-POC',\n",
+       " 'Ed-Fi-Exchange-OSS/EdFi-Project-Buzz',\n",
+       " 'Ed-Fi-Exchange-OSS/CloudODS',\n",
+       " 'Ed-Fi-Exchange-OSS/Ed-Fi-X-DescriptorTool',\n",
+       " 'Ed-Fi-Exchange-OSS/LMS-Toolkit',\n",
+       " 'Ed-Fi-Exchange-OSS/Ed-Fi-Solution-Scripts',\n",
+       " 'Ed-Fi-Exchange-OSS/Student360',\n",
+       " 'Ed-Fi-Exchange-OSS/Tableau-Attendance-Dashboard',\n",
+       " 'Ed-Fi-Exchange-OSS/Ed-Fi-MetabaseChronicAbsenteeismQuickStart',\n",
+       " 'Ed-Fi-Exchange-OSS/Ed-Fi-Sample-Data-Time-Travel-Script',\n",
+       " 'Ed-Fi-Exchange-OSS/DataChecker',\n",
+       " 'Ed-Fi-Exchange-OSS/Ed-Fi-TPDM-Community-Artifacts',\n",
+       " 'Ed-Fi-Exchange-OSS/Chronic-Absentee-Reports-Tableau',\n",
+       " 'Ed-Fi-Exchange-OSS/Family-Engagement-Portal',\n",
+       " 'Ed-Fi-Exchange-OSS/Analytics-Middle-Tier-Contrib',\n",
+       " 'Ed-Fi-Exchange-OSS/Ed-Fi-BinariesPowerShellInstaller',\n",
+       " 'Ed-Fi-Exchange-OSS/Template-for-GitHub',\n",
+       " 'Ed-Fi-Exchange-OSS/PowerShell-NuGet-Installer',\n",
+       " 'Ed-Fi-Exchange-OSS/EPP-PowerBI-Report-Starter-Kit',\n",
+       " 'Ed-Fi-Exchange-OSS/Ed-Fi-Clever-Integration',\n",
+       " 'Ed-Fi-Exchange-OSS/Ed-Fi-Roster-Sample-Application',\n",
+       " 'Ed-Fi-Exchange-OSS/Ed-Fi-Google-Classroom-Roster',\n",
+       " 'Ed-Fi-Exchange-OSS/Ed-Fi-to-Generate']"
+      ]
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from edfi_tech_metrics.ossf_score import fetch_repositories, report_ossf_scores\n",
+    "\n",
+    "# Fetch repositories for Ed-Fi-Alliance-OSS and Ed-Fi-Exchange-OSS\n",
+    "ed_fi_alliance_repos = fetch_repositories('Ed-Fi-Alliance-OSS')\n",
+    "ed_fi_exchange_repos = fetch_repositories('Ed-Fi-Exchange-OSS')\n",
+    "\n",
+    "# Combine the repositories\n",
+    "repositories = ed_fi_alliance_repos + ed_fi_exchange_repos\n",
+    "\n",
+    "# Display the repositories\n",
+    "repositories"
```

Comment:
This is useful for debugging, but not very useful to the reader. I recommend commenting it out.